docs/en/resources/tools/bigquery/bigquery-forecast.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "bigquery-forecast"
+type: docs
+weight: 1
+description: >
+  A "bigquery-forecast" tool forecasts time series data in BigQuery.
+aliases:
+- /resources/tools/bigquery-forecast
+---
+
+## About
+
+A `bigquery-forecast` tool forecasts time series data in BigQuery.
+It's compatible with the following sources:
+
+- [bigquery](../../sources/bigquery.md)
+
+`bigquery-forecast` constructs and executes a `SELECT * FROM AI.FORECAST(...)` query based on the provided parameters:
+
+- **history_data** (string, required): This specifies the source of the historical time series data. It can be either a fully qualified BigQuery table ID (e.g., my-project.my_dataset.my_table) or a SQL query that returns the data.
+- **timestamp_col** (string, required): The name of the column in your history_data that contains the timestamps.
+- **data_col** (string, required): The name of the column in your history_data that contains the numeric values to be forecasted.
+- **id_cols** (array of strings, optional): If you are forecasting multiple time series at once (e.g., sales for different products), this parameter takes an array of column names that uniquely identify each series. It defaults to an empty array if not provided.
+- **horizon** (integer, optional): The number of future time steps you want to predict. It defaults to 10 if not specified.
```

Comment:
What types of questions can you use with the agent to get these inputs accurate? And what types of error messages are you getting when the inputs are wrong?

Reply:
Manual testing showed this is fine.

Reply:
I added some sample prompts. Thanks!